config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -64,6 +64,7 @@ presubmits:
     path_alias: sigs.k8s.io/kind
     decoration_config:
       timeout: 40m
+      terminationGracePeriodSeconds: 300
```

Comment:
this is wrong, bad cut and paste, it's `grace_period: 15s` (for the default anyhow)

Reply:
`terminationGracePeriodSeconds`  is the pod.spec field, right?